### ISSUE TITLE
3DSSG

### ISSUE THREAD

**Issue by linukc** [linukc/SUN3D_DATASETS#6]
3DSSG provides 3D semantic scene graphs for [3RScan](https://github.com/3DSSG/3DSSG.github.io/blob/master/waldjohannau.github.io/RIO). A semantic scene graph is defined by a set of tuples between nodes and edges where nodes represent specific 3D object instances in a 3D scan. Nodes are defined by its semantics, a hierarchy of classes as well as a set of attributes that describe the visual and physical appearance of the object instance and their affordances. The edges in our graphs are the semantic relationships (predicates) between the nodes such as standing on, hanging on, more comfortable than or same material.
In summary, our dataset features 1482 scene graphs with 48k object nodes and 544k edges.

[Paper](https://arxiv.org/pdf/2004.03967) [Project](https://3dssg.github.io/) [Code](https://github.com/ShunChengWu/3DSSG)

```
  @inproceedings{3DSSG2020,
    title={Learning 3D Semantic Scene Graphs from 3D Indoor Reconstructions},
    author={Wald, Johanna and Dhamo, Helisa and Navab, Nassir and Tombari, Federico},
    booktitle={Conference on Computer Vision and Pattern Recognition (CVPR)}, 
    year={2020}
  }
```
